test-end-to-end/android-maestro/e2e-tests/003_002_add_module_text_blocks_and_perform_indent_hide_delete_actions.yaml

```diff
@@ -0,0 +1,31 @@
+appId: com.toughra.ustadmobile
+env:
+  TESTRESULTSDIR: "${TESTRESULTSDIR}/003_002_add_module_text_blocks_and_perform_indent_hide_delete_actions"
+  TESTNAME:  "003_002_add_module_text_blocks_and_perform_indent_hide_delete_actions"
+---
+- clearState: com.toughra.ustadmobile
+- runFlow:
+    file: "../common/startserver.yaml"
+    env:
+      CONTROLSERVER: ${CONTROLSERVER}
+      TESTSERIAL: ${TESTSERIAL}
+      TESTNAME: ${TESTNAME}
+
+- runFlow: "../common/clear_state_and_get_started.yaml"
+- runFlow: "../common/select_site.yaml"
+#admin login
+- runFlow:
+    file: "../common/user_login.yaml"
+- runFlow:
+    file: "../common/add_course.yaml"
+    env:
+     TESTCOURSENAME: "Test Course Block Test"
+#--- add module and text block, test actions-hide/unhide/indent/unindent/delete ----#
+- runFlow:
+    file: "../common/block_indent_hide_delete.yaml"
```

Comment:
Is block_indent_hide_delete used in any other flows? If not, then we don't need to have it in common. It can just be directly here.

Reply:
Yes, that's done. Thanks Mike